scripts/build-rancherd

```diff
@@ -31,3 +31,5 @@ cd ../..
 ./scripts/chart/package
 
 sed -e "s|%{CHART_CONTENT}%|$(base64 -w0 ./bin/chart/*/rancher-${CHART_VERSION}.tgz)|g" ./scripts/rancher.yaml.tmpl > ./bin/rancher.yaml
+cp ./scripts/rancher-namespace.yaml.tmpl ./bin/rancher-namespace.yaml
```

Comment:
is it going to surprises someone later on down the road that you're just copying this template file as-is instead of actually templating it?

Reply:
Yeah, Just following the same naming conversion in case we need to template. Think It should be fine in here :)